### ISSUE TITLE
Quiz for New Volunteers to Help w/ Team Selection

### ISSUE THREAD

**Issue by sherririllingux** [hackforla/website#3005]
### Overview
We need to create a quiz for new UX Research volunteers who are going through onboarding, that can help them assess whether they are interested in and have the skills to lead a team or whether they'd be better suited for a role within the team, not in leadership.

### Action Items

- [x] Review **Communities of Practice** on Github
- [x] Review **Developer Self-Test** as an example to understand a format that could work for this quiz
- [x] Determine what teams are available to join and create a list. Look for teams that are seeking Leads as well.
- [x] Create quiz that incorporates the below information. Feel free to break quiz into 2 or more slides as done in the Developer Self-Test, for optimal accuracy. Come up with a few questions (you'll find statement-question examples below) to narrow down what team and role the new volunteer should join
   - [x] "I feel comfortable and confident..."
      - [x] Taking work and breaking it into smaller tasks so that multiple people can work on it
      - [x] Organizing and leading meetings
      - [x] Googling examples when I don't have the answer
      - [x] Reviewing content/examples from other teams (when provided) to replace a successful process
      - [x] Creating guidance for others after I have learned how to do something myself
      - [x] Collaborating/discussing ideas with other team Leads and/or other team members to create the best product for the user
      - [x] Communicating on behalf of others (Team Representative)
      - [x] Giving constructive feedback to teammates
      - [x] Providing an overview to a new team member
   - [x] "I know what the following are and have a basic understanding of how to execute that task"
      - [x] Personas
      - [x] Interviews
      - [x] Surveys
      - [x] Usability Testing
      - [x] A/B Testing
      - [x] Wireframe Sketches
      - [x] Using Figma
      - [x] Qualitative Analysis
      - [x] Quantitative Analysis
      - [x] Accessibility Standards
- [x] Create a structure with the items above (or questions you come up with) like the ones in the Developer Self-Test and a link at the end directing to the projects/teams available to suggest teams for new volunteers .
- [x] Review quiz with Lead and Product team to gain feedback and make any necessary changes.
- [x] Submit quiz to Bonnie

### Resources/Instructions

- [Communities of Practice](https://github.com/hackforla/communities-of-practice) 
- [Developer Self-Test](https://docs.google.com/presentation/d/1bueC40PkklLSJVPttiFeYJvcB69PgJx8pxZK5G7gzzo/edit#slide=id.gf938725143_0_20)
- [UXR Quiz V.1 9/18/22](https://docs.google.com/presentation/d/1qliHe6GX-y3J_djoYBfyQNNeFYn2n-swu7tYvSU7ZTk/edit?usp=sharing)

**Comment by ExperimentsInHonesty** [comment]
@sherririllingux - I can't evaluate this issue until I understand what the recommendations that led to this being added to the spreadsheet are.  The action required was to look at the research and determine what exactly was being asked.  At this point, we don't even know if an issue is required.

> What does this mean?  Look at research to determine.  [See Developer self test in new onboarding deck](https://docs.google.com/presentation/d/1bueC40PkklLSJVPttiFeYJvcB69PgJx8pxZK5G7gzzo/edit#slide=id.g10c55e2c17c_0_30)  Is this what is meant?

**Comment by sherririllingux** [comment]
I have not been able to find data backing up the need for a volunteer quiz. I am currently waiting on access to Google Docs to possibly find information about why a quiz is needed, but looking through the wiki pages, there is nothing that backs up the [action step](https://docs.google.com/spreadsheets/d/1NjrbicVDyd8aTXfPXUAdlX2xg9F3Ez9VThj4EwG-ouw/edit#gid=0) for a quiz. The action steps do specify to look at the recommendations presentation, and I believe that is in Google Docs so I will check there once I get access.

**Comment by ExperimentsInHonesty** [comment]
I was the instigator for this issue.  The problem we are trying to solve is that during onboarding, people don't know which project to choose.  It's easy for the developers because we do it based on experience, and HfLA website is a level 1 development team, and the other hackforla website development teams are level 2, which means there is less of a guided path.

**Comment by ExperimentsInHonesty** [comment]
User Research methods: 
- personas
- interviews
- surveys
- user testing of mockups
- usability testing of working product

Leading a team
Qualitative Analysis (Affinity mapping)
Quantitative Analysis
Accessibility standards

**Comment by ExperimentsInHonesty** [comment]
User Researcher and Research Lead

Statements
I feel comfortable :
1. Taking work and breaking it into smaller tasks so that multiple people can work on it
1. Organizing and Leading meetings
1. Googling examples when I don't have the answer
1. Reviewing content/examples from other teams (when provided) to replace a successful process
1. Creating Guidance for others after I have learned how to do something myself
1. Communicating on behalf of others (team representative)
1. Giving constructive feedback to teammates
1. Work collaboratively with multiple people in a meeting
1. Providing an overview to a new team member

I know what the following things are:

1. personas
1. interviews
1. surveys
1. user testing of mockups
1. usability testing of working product
1. Qualitative Analysis
1. Quantitative Analysis
1. Accessibility standards

**Comment by ExperimentsInHonesty** [comment]
Program Designer
Defining an overarching strategy for all the research (research roadmap)

Instructional Design
Making slides that teach people how to do the research methods for the specific tasks (ie., affinity mapping, coding, etc.)

**Comment by bradyse** [comment]
Hi @sacamp ,

@ExperimentsInHonesty asked me to review this issue to determine if the UXR quiz is ready for onboarding. The tl;dr version of my review: There are too many "I feel comfortable statements" that are not specifically related to UXR. Knowing about the 8 skills listed is great but asking people if they have previous experience is better. I recommend onboardees self-score their responses.

Thanks!
Sara

### "I feel comfortable" statements
My initial impressions is that The "I feel comfortable" statements are not entirely specific to UXR. Some of them could be relevant for other roles. For example, giving constructive feedback to teammates would also be applicable to a UX designer, data scientist, project manager, developer, or really anyone. When paired with the skills questions, I could see these items being useful. I would recommend adding the word "very" before comfortable to really hammer home the point that to be effective in these roles, people need to feel very comfortable doing these tasks.

### UXR Skills Checklist
For the 8 skills listed, Bonnie seems to have revised the question stem to say "I know what the following things are:". I think that knowing what something is is different from being able to execute or perform the skills (as what is listed in the GitHub action item).  I like Bonnie's revision because it removes the double-barreled ambiguity of the original question stem, but I recommend adding a follow-up question that gets at whether people actually have experience with performing each skill they know about.

For example, I know what qualitative analysis is, but I have never actually done one by myself. Furthermore, I couldn't tell you off the top of my head what is the difference between an ethnography or grounded theory qualitative analysis. Therefore, I recommend that you ask onboardees if they have had previous experience executing the skills that they know about it. 

### Quiz Implementation
I recommend that onboardees are given a scoring system so that whoever is running the onboarding doesn't have to triage with people directly on UXR placement.

Something like:

Give yourself 1 point for each skill you know about.
Give yourself 1 point for each skill that have had prior experience.
Give yourself 1 point for each team statement you feel comfortable with.

If you scored above X, you are qualified to take a UXR lead role.
If you scored at or below X, you are qualified to a UXR non-lead role.

**Comment by vzpeopledesign** [comment]
**Progress**: Currently reviewing the first draft of the UXR quiz.
**Blockers**: 1. The editing process  2. Figure out why "Determine what teams are available to join and create a list. Look for teams that are seeking Leads as well." is necessary when a link to the project page can be provided.
**Availability**: I have 6 hours availability Fri/Sat and then 2 hours availability Sunday
**ETA**: Quiz will be ready for submission to Bonnie no later than 9/4/2022.

Below is the first draft of quiz for review:

Hello fellow UXers! Below is the first working draft of the UX research leadership quiz.

Goal: Funnel new members into the right role: leadership or team member

Rationale behind the format: Following the steps from predecessors, the quiz is meant to quantify how qualified the new member is as a UX researcher and as a leader. I decided to assess their UX research knowledge and funnel them to a team member role if their knowledge wasn’t that strong. If knowledgeable, then assess their leadership capability and funnel them at that point as well. I reasoned that knowledge of the field is more critical than the leadership skills for the projects at Hack for LA given the time schedule is more lax and therefore more forgiving of less leadership experience.

The Scoring: For Slide 2, I reasoned that we need someone who is at least familiar with all research tasks for a leadership role with some experience in most categories, hence 16/20 points. I applied the same rationale to Slide 3, with 25 points and above qualifying for a leadership role.

Feedback:
-Does the order of questioning work? UX knowledge vs. leadership assessments
-Assessing UX research knowledge: was anything left out? Anything you’d consider redundant?
-Assessing leadership experience: was anything left out? Anything you’d consider unnecessary?
-Will the point system accurately funnel newcomers to the appropriate projects?

Now onto the first draft of the UX research onboarding quiz! 

[SLIDE 1]
User Researcher and Research Lead
Welcome to UX Research at Hack for LA! We know onboarding is a long process, but you are almost done. Congrats! To help you find your way to the role that best suits you take the quiz below to see if you are a fit for a UX research lead role or UX researcher role.
[SLIDE 2]
For the following give yourself:
1 point for each UX method you are familiar with, 2 points for each method you have at least some experience with.

Personas
User interviews
User surveys
User testing of mockups
A/B testing
Card sorting
Usability testing of working product
Qualitative Analysis
Quantitative Analysis
Accessibility standards

Now, add your points up. If you have more than 16 points, go to the next slide. If you have 16 points or less, click on the link below to find an open project to join. Good luck!
[UX RESEARCH OPEN PROJECTS LINK]
[SLIDE 3]
For the following give yourself: 
1 point if you are familiar, 2 points if you have some experience, and 3 points if you feel comfortable

Taking work and breaking it into smaller tasks so that multiple people can work on it
Organizing and leading meetings
Troubleshooting any blockers the team may have
Maintaining a realistic pace on projects
Googling examples when I don't have the answer
Reviewing content/examples from other teams (when provided) to replace a successful process
Creating Guidance for others after I have learned how to do something myself
Communicating on behalf of others (team representative)
Giving constructive feedback to teammates
Work collaboratively with multiple people in a meeting
Providing an overview to a new team member, getting them started

Now, add your points up. If you have more than 25 points, then a leadership role may be a better fit for you. If interested in a UX lead researcher role, click on the link below and find a team project you are interested in leading. 
[UX RESEARCH LEAD - OPEN PROJECTS LINK]

If you are not interested in a leadership role, there are plenty of UX research projects that need your help! Go here to explore open projects. 
[UX RESEARCH OPEN PROJECTS LINK]

Good luck researchers!
_________________________________________________________________________



Thank you for reviewing!

**Comment by vzpeopledesign** [comment]
Progress: I need to implement the changes suggested by Bonnie and present the product to her
Blockers: None at this moment
Availability: I have the next 2 weeks until 9/11
ETA: 9/8

Bonnies notes:

Two slides 
read the list of items and give yourself 0.5 point if you have familiarity and a point if you have done it.
Slide 1 UX skills
Slide 2 Management Skills
Write your score on your post it
As slide1/slide2 (e.g., 5/10)

**Comment by vzpeopledesign** [comment]
Update: Putting together the UX Researcher Quiz slides for Bonnie's review, slides created will be based on the above outline 
Ready by: 9/18/2022

**Comment by vzpeopledesign** [comment]
Final result now part of the Onboarding Slide deck pegs 23-25. Project completed. May need follow up if the parameters of choosing a UXR leader aren't working.

**Comment by HackforLABot** [comment]
Hi @vzpeopledesign, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)
